### PR TITLE
Package moves

### PR DESCRIPTION
**Pull request by joolswills**
Some of these have been discussed in #2961 

Ive done it slightly differently but some of these are better defaults.

Im considering moving all n64 to optional also. Will make it easier for users to make a considered choice imho.

### CODE REVIEW

File: scriptmodules/emulators/mupen64plus.sh [hunk 1]
```diff
@@ -173,16 +173,14 @@ function install_mupen64plus() {
...
+    isPlatform "kms" && res="%XRES%x%YRES%"
 
     if isPlatform "rpi"; then
-        # mesa needs to run at full screen as it doesn't benefit from our SDL scaling hint
-        if isPlatform "mesa"; then
-            res="${resolutions[0]}"
-            addEmulator 1 "${md_id}-GLideN64" "n64" "$md_inst/bin/mupen64plus.sh mupen64plus-video-GLideN64 %ROM% $res 0 --set Video-GLideN64[UseNativeResolutionFactor]\=1"
+        # kms needs to run at full screen as it doesn't benefit from our SDL scaling hint
+        if isPlatform "kms"; then
+            addEmulator 0 "${md_id}-GLideN64" "n64" "$md_inst/bin/mupen64plus.sh mupen64plus-video-GLideN64 %ROM% $res 0 --set Video-GLideN64[UseNativeResolutionFactor]\=1"
```
Comment (dankcushions) [inline scriptmodules/emulators/mupen64plus.sh]: I think you want this one with `addEmulator 1` as it’s the best default for kms

**Comment by joolswills** [comment]
It's disabled in favour of lr-mupen64plus-next as requested.

File: scriptmodules/emulators/mupen64plus.sh (hunk 1 above)
Comment (joolswills) [reply scriptmodules/emulators/mupen64plus.sh]: I just forgot to make that default also.

Comment (joolswills) [reply scriptmodules/emulators/mupen64plus.sh]: I've done that now.

Comment (dankcushions) [reply scriptmodules/emulators/mupen64plus.sh]: in https://github.com/RetroPie/RetroPie-Setup/issues/2961 i suggested mupen64plus standalone stays as default. it should be more performant/compatible as i think the libretro version isn't threaded yet, and may have some bugs that standalone does not.

File: scriptmodules/libretrocores/lr-snes9x.sh [hunk 2]
```diff
@@ -41,6 +41,8 @@ function configure_lr-snes9x() {
     mkRomDir "snes"
     ensureSystemretroconfig "snes"
 
+    local def=0
+    ! isPlatform "armv6" && ! isPlatform "armv7" && def=1
     addEmulator 0 "$md_id" "snes" "$md_inst/snes9x_libretro.so"
```
Comment (dankcushions) [inline scriptmodules/libretrocores/lr-snes9x.sh]: does this want to be `addEmulator $def` rather than `addEmulator 0`?

Comment (joolswills) [reply scriptmodules/libretrocores/lr-snes9x.sh]: It does yep.

**Comment by dankcushions** [comment]
thumbs up for lr-snes9x. just gave it a quick test and it seems to stand up to runahead = 1, and threaded video = disable, which are two big latency reducers.

File: scriptmodules/emulators/mupen64plus.sh (hunk 1 above)
Comment (joolswills) [reply scriptmodules/emulators/mupen64plus.sh]: I'm not convinced. I may switch it.

**Pull request merged**